Alphabet-to-LB_readme.md

```diff
@@ -20,12 +20,14 @@ This Python script removes punctuation, converts upper to lower-case letters, ha
 
 Currently, this script is only made available as a code sample.  Many points of linguistic complexity are elided, and there is still some room to improve the accuracy of the script when it comes to acronyms and foreign words.  Please don't publish with this script!  You have been warned.
 
-## Config
+## Instructions for Use

```

Comment:
I recommend including a single copy-pastable line like this:
```
python Alphabet-To-LB.py
```

I recommend putting it in triple backticks, like so:

\``` 
python python Alphabet-To-LB.py
\``` 

That will create the code block effect you see in my example, which causes the code stand out visually.

Reply:
I'm not sure what you intend-- where should I place the single line?

Reply:
Immediately after the `Instructions for Use` line. It should be the first thing in this section.

Like this, but with triple backticks around the `python Alphabet-To-LB.py` line so that it appears in a code block:
```
## Instructions for Use

python Alphabet-To-LB.py

For language-specific processing, Alphabet to Linear B takes a configuration file-- a Python script which eliminates many-to-one and one-to-many mappings between script and language, and also outlines the language's orthography and phonology.  Alphabet to Linear B also takes a .txt file containing the section of text you would like converted.  Config files and data sets have been provided for Turkish and German.

Upon running the script, you will be prompted for the name of the language-specific config file.  The config file will contain the name of the .txt file you wish to convert, so no need for anything else.  Sit back and enjoy!  You will see notifications when the input file is processed and the output file (OutputFile.txt) is written.  Current run time stands at around 10-15 minutes for 10k words, which is the size of the files provided as examples.
```